Applications/CLI/ogs.cpp

```diff
@@ -92,9 +92,9 @@ int main(int argc, char *argv[])
 	cmd.add(project_arg);
 	cmd.parse(argc, argv);
 
-	ApplicationsLib::LogogSetup logog_setup;
 	ApplicationsLib::LinearSolverLibrarySetup linear_solver_library_setup(
 	    argc, argv);
+	ApplicationsLib::LogogSetup logog_setup;
```

Comment:
Yes, MPI/PETSc initialization is required before  logog_setup, which needs rank and rank size of MPI.